From 82a58327b548c35df24b312baf73f2a9e451dc35 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Sat, 24 Mar 2018 13:20:52 -0400
Subject: [PATCH 11/35] camera: Check if aux camera whitelist is set before
 restricting cameras

Some devices have a fully featured 3rd camera and adding 20+
camera apps to the whitelist is not just stupid, it's impossible
due to the string length limit on systemprops. Add a check to see
if the prop is even set, and if not, just restore the original
behaviour and expose all the cameras to all the apps.

Change-Id: I6c3b33c077e8710c73b5d0fa28e1b017d6c43a58
---
 core/java/android/hardware/Camera.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/java/android/hardware/Camera.java b/core/java/android/hardware/Camera.java
index 07893238eef..e56836bff0e 100644
--- a/core/java/android/hardware/Camera.java
+++ b/core/java/android/hardware/Camera.java
@@ -288,8 +288,9 @@ public class Camera {
                     return true;
                 }
             }
+            return false;
         }
-        return false;
+        return true;
     }
 
     /**
-- 
2.17.1

